src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -37,7 +38,7 @@ public void execute() {
             m_translationXSupplier.getAsDouble(),
             m_translationYSupplier.getAsDouble(),
             m_rotationSupplier.getAsDouble(),
-            m_drivetrainSubsystem.getGyroscopeRotation()));
+            m_drivetrainSubsystem.getGyroscopeRotation().plus(Rotation2d.fromDegrees(90))));
```

Comment:
What is causing us to want to add 90 degrees?

Reply:
Adding 90 degrees here allows to zero the gyroscope the same way he did before the zeroGyroscope change (I think you noticed that already but still)